s2n-tls.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: s2n-tls
-  version: 1.4.18
+  version: 1.4.19
```

Comment:
wonder why the expected commit hasn't changed in this PR 🤔

Reply:
It looks like the tag was deleted upstream - see https://github.com/aws/s2n-tls/tags and issue https://github.com/aws/s2n-tls/issues/4673 filed to track 1.4.19 build issues. 

There is currently no diff in upstream between 1.4.18 and main - see https://github.com/aws/s2n-tls/compare/main...v1.4.18

As such. Reject.